suite/tests/CMakeLists.txt

```diff
@@ -3737,6 +3737,28 @@ if (BUILD_CLIENTS)
     set(tool.drcacheoff.windows-split_postcmd3
       "firstglob@${drcachesim_path}@-indir@${dir_prefix}.*.dir/raw/window.0002@-simulator_type@basic_counts")
 
+    # Test L0_filter_until_instrs
+    torunonly_drcacheoff(L0_filter_until_instrs-windows-split ${ci_shared_app}
```

Comment:
Would be better to s/L0_filter_until_instrs/L0-filter-until-instrs/ for consistent use of '-' in the test name.

 Same for tests below.

Reply:
I was thinking of L0_filter_until_instrs as "one word" and the hyphens to be demarcating words but I wasn't consistent. I have changed it to only use hyphens.